### PR TITLE
Fix issue #209 and #210

### PR DESCRIPTION
**Pull request by Firesphere**
# Please check the following

- [x] Have you opened an issue to discuss the feature and agree its general design?
- [x] Do you have a use case and, ideally, an example program using the feature?
- [x] Do you have tests covering 90%+ of the feature code (and, of course passing)
- [x] Have you written complete and accurate doc comments?
- [x] Have you updated the README or docs where needed?
- [x] Code is readable
- [x] Code is tested
- [x] I've read the Code of Conduct (Or, at least, I do believe I agree with it)
- [x] Frontend features don't look completely uncared for
- [x] Code is PSR-2 compliant
- [x] I'm fairly sure this is useful
- [x] I'm not just making this PR for HacktoberFest
- [x] You rock. Thanks a lot.

### CODE REVIEW

File: tests/unit/BaseIndexTest.php [hunk 1]
```diff
@@ -308,6 +308,21 @@ public function testDoSearch()
...
+        $index->doSearch($query);
+
+        $this->assertEquals(['*:*'], $index->getQueryTerms());
+
+        $index = new CircleCITestIndex();
+        $query = new BaseQuery();
+        $query->addTerm('test');
+        $query->addSort('SiteTree_Title', 'asc');
+        $index->doSearch($query);
+
```
Comment (ssmarco) [inline tests/unit/BaseIndexTest.php]:
```suggestion
$this->assertEquals(['test'], $index->getQueryTerms());
```

Comment (Firesphere) [reply tests/unit/BaseIndexTest.php]: It's a bit a double-up from queries tested above, but indeed, it never hurts to be thorough.

**Comment by kinglozzer** [comment]
Nice, thanks Simon!

**Comment by Firesphere** [comment]
You're welcome, keep those issues/pr's coming @kinglozzer !

**Pull request merged**